src/main/java/com/spotinst/sdkjava/example/statefulnode/azure/AzureStatefulNodeExample.java

```diff
@@ -0,0 +1,558 @@
+
+    public static void main(String[] args) throws InterruptedException {
+
+        SpotinstAzureStatefulNodeClient nodeClient = SpotinstClient.getAzureStatefulNodeClient(auth_token, act_id);
+
+        //Create Stateful Node
+        System.out.println("----------Creation of azure stateful node--------------");
+        String nodeId = createStatefulNode(nodeClient);
+
+        //Get Stateful Node
+        System.out.println("----------Get Stateful Node--------------");
+        getStatefulNode(nodeClient, nodeId);
+
+        //Update Stateful Node
+        System.out.println("----------Update Stateful Node Name--------------");
+        updateStatefulNodeName(nodeClient, nodeId);
+
+        //update Stateful Node Strategy
+        System.out.println("----------Update Stateful Node Strategy--------------");
+        updateStatefulNodeStrategy(nodeClient, nodeId);
+
+        //update Stateful Node Data Disk
+        System.out.println("----------Update Stateful Node DataDisk--------------");
+        updateStatefulNodeDataDisk(nodeClient, nodeId);
+
+        //update Stateful Node Os Disk
+        System.out.println("----------Update Stateful Node OsDisk--------------");
+        updateStatefulNodeOsDisk(nodeClient, nodeId);
+
+        //update Stateful Node Scheduling Tasks
+        System.out.println("----------Update Stateful Node Scheduling--------------");
+        updateStatefulNodeScheduling(nodeClient, nodeId);
+
+        //update Stateful Node Health
+        System.out.println("----------Update Stateful Node Health--------------");
+        updateStatefulNodeHealth(nodeClient, nodeId);
+
+        //update Stateful Node Persistent
+        System.out.println("----------Update Stateful Node Persistent--------------");
+        updateStatefulNodePersistent(nodeClient, nodeId);
+
+        //Get All Stateful Node
+        System.out.println("----------List All Stateful Nodes--------------");
+        getAllStatefulNode(nodeClient);
+
+        //Recycle Stateful Node
+        System.out.println("----------Recycle Stateful Node--------------");
+        recycleStatefulNode(nodeClient, nodeId);
+
+        //Get Stateful Node Status
+        System.out.println("----------Get Stateful Node Status--------------");
+        getStatefulNodeStatus(nodeClient, nodeId);
+
+        //Pause Stateful Node
+        System.out.println("----------Pause Stateful Node--------------");
+        pauseStatefulNode(nodeClient, nodeId);
+
+        //Pause Stateful Node
+        System.out.println("----------Resume Stateful Node--------------");
+        resumeStatefulNode(nodeClient, nodeId);
+
+        //Delete Stateful Node
+        System.out.println("----------Delete Stateful Node--------------");
+        deleteStatefulNode(nodeClient, nodeId);
+
+    }
+
+    private static String createStatefulNode(SpotinstAzureStatefulNodeClient client) {
```

Comment:
Change the example to use the Automation Resources

Reply:
As Discussed, will not change here in example file.